lib/paperclip/attachment.rb

```diff
@@ -538,6 +547,19 @@ def post_process_style(name, style) #:nodoc:
       end
     end
 
+    def generate_styles(generator, styles)
+      style_options = {}
+      styles.reject { |name, style| name == :original }.each do |name, style|
```

Comment:
Unused block argument - `style`. If it's necessary, use `_` or `_style` as an argument name to indicate that it won't be used.